source/adios2/toolkit/format/dataman/DataManSerializer.tcc

```diff
@@ -200,7 +200,7 @@ int DataManSerializer::GetData(T *outputData, const std::string &varName,
                                const size_t step, const Dims &varMemStart,
                                const Dims &varMemCount)
 {
-    PERFSTUBS_SCOPED_TIMER_FUNC();
```

Comment:
Grepping through the code, everywhere else the invocation of the macro PERFSTUBS_SCOPED_TIMER_FUNC() has a trailing semicolon.  It's a null definition unless we build with Tau (which we don't in CI), but I worry that this change might break something.  At the very least it's unnecessary.  Revert?

Reply:
I think it is not needed everywhere, but I can restore it.

Reply:
Well, it was there for a reason, not a functionality reason but for performance diagnostics.  Those aren't maintained as reliably they they probably should be, but killing an existing monitoring point in an unrelated change is not a step in the right direction...